### DISCUSSION TITLE
Why can't this addon support ingress?

### DISCUSSION THREAD

**Discussion by leonardochaia** [hassio-addons/addon-nginx-proxy-manager#339] [Q&A]
Hello all,
I am a very curious person..

> This add-on cannot support it, therefore closing the issue.

@frenck [[source]](https://github.com/hassio-addons/addon-nginx-proxy-manager/issues/12#issuecomment-487197059)

@frenck would you clarify this statement a bit more?
What is the limitation that prevent this addon from using ingress?
I understand you have stated multiple times that it cannot be supported but I could not find the reasons why.

Thank you all for such a great addition to HA.

Leo.

**Comment by frenck** [comment]
Hi there @leonardochaia 👋 

At the time, the admin interface of the NGINX Proxy Manager didn't play with being served from a reverse proxy that puts it in a subfolder (which is basically what Ingress does).

It held on to its root folder hardcoded in many places (thus the frontend hitting Home Assistant instead of the add-on for its files).

Things might have changed over time, I haven't really investigated again ever since.

../Frenck

**Reply by leonardochaia** [reply]
Hi, appreciate the prompt reply :)

Yeah that makes sense (Y)
>  (thus the frontend hitting Home Assistant instead of the add-on for its files).

It's funny how a proxy manager cannot run behind a proxy lol. I'm gonna check the issues over their repo to see if I find something relevant.

Thanks for the reply @frenck have a great day :+1:

**Comment by Wheemer** [comment]
Ingress would sure be nice.

**Comment by Silther** [comment]
I have setup a new domain for NGINX Proxy Manager [npm.example.com](), with which I can directly open NGINX, but if I try to open it though home Assistant (web ui buttoon) it opens [homeassistant.example.com:81]() instead.
Is there a way to redirect [homeassistant.example.com:81]() to [npm.example.com]()

**Reply by frenck** [reply]
No there is not.
